server/routes/createGoal/createGoalController.originalBehaviour.ts

```diff
@@ -0,0 +1,162 @@
+import createError from 'http-errors'
+import type { RequestHandler } from 'express'
+import type { NewGoal } from 'compositeForms'
+import moment from 'moment'
+import EducationAndWorkPlanService from '../../services/educationAndWorkPlanService'
+import CreateGoalView from './createGoalView'
+import AddStepView from './addStepView'
+import AddNoteView from './addNoteView'
+import { toCreateGoalDto } from './mappers/createGoalFormToCreateGoalDtoMapper'
+import validateCreateGoalForm from './createGoalFormValidator'
+import validateAddStepForm from './addStepFormValidator'
+import ReviewView from './reviewView'
+import futureGoalTargetDateCalculator from '../futureGoalTargetDateCalculator'
+
+/**
```

Comment:
This new class is a verbatim copy and paste of the original `createGoalController`  but with this comment block and a different class name. Functionally it's identical to the original.